schemas/input/csl-data.json

```diff
@@ -243,7 +243,17 @@
             "alternate": {
               "title": "Alternative name, such as screen name for online item or real name of pseudonymous author.",
               "description": "E.g. rendered as 'Smith, J. [@JSmith]'",
-              "type": "string"
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/name-variable"
+              }
+            },
+            "place": {
+              "title": "Place associated with the name",
+              "type": ["string", "array"],
+              "items": {
```

Comment:
I'm not certain, but I don't think this is right. You may need to use `oneOf` instead.

Reply:
Also, annotations?

Reply:
I think you can have an array of allowed types. Restrictions are only applied to the types they are valid for (here, "items" is applied to "array" but not to "string" type values). See https://cswr.github.io/JsonSchema/spec/multiple_types/

We have `["string", "number"]` several places in the schema already.